lib/server.js

```diff
@@ -225,6 +242,13 @@ module.exports = function createServer(config, log) {
               retryAfter: limits.rateLimitIntervalSeconds,
               unblock: false
             })
+
+            if (config.reputationService.enable) {
```

Comment:
On server side errors, should it be reported to your service as a blocked IP? Seems like a grey area because you could potentially penalize users for an error that was not caused by them.